.github/workflows/test.yml

```diff
@@ -7,7 +7,7 @@ jobs:
     strategy:
       matrix:
         os: ['ubuntu-latest', 'macos-latest', 'windows-latest']
-        node: [12, 14, 16, 18]
+        node: [14, 16, 18]
```

Comment:
Dropping node 12 from the pipeline because one of the Volar packages is incompatible. Not really dropping Node 12 support in general though, as it will still work with Node 12 with non-Vue projects. Thus this won't need a major bump.